src/client/components/MessageForm/SendMessageForm.css

```diff
@@ -0,0 +1,15 @@
+footer {
+  background-color: lightgray;
+  color: #ffffff;
+  padding: 10px 0;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+}
+footer input {
+  margin-left: 10px;
+  width: 1000px;
+}
+footer img {
+  width: 30px;
+}
```

Comment:
The message form is not a footer, just a form

Reply:
I used as a form.Thank you.